### PR TITLE
Add PMM to broker

### PR DESCRIPTION
**Pull request by nonemax**

### CODE REVIEW

File: broker/cluster.go [hunk 1]
```diff
@@ -41,6 +41,13 @@ func (p *Controller) DeployCluster(instance ServiceInstance, skipS3Storage *bool
 		}
 		conf.PXC.BrokerInstance = string(brokerInstance)
 
+		if instance.Parameters.PMM.Enabled {
+			conf.PMM.Image = instance.Parameters.PMM.Image
+			conf.PMM.ServerHost = instance.Parameters.PMM.Host
+			conf.PMM.ServerUser = instance.Parameters.PMM.User
```
Comment (cap1984) [inline broker/cluster.go:47]: conf.PMM = &instance.Parameters.PMM ?

**Review by cap1984 — COMMENTED** [review]
What about operator version?

File: broker/cluster.go (hunk 1 above)
Comment (nonemax) [reply broker/cluster.go:47]: different types

File: broker/service-instance.go [hunk 2]
```diff
@@ -76,3 +77,11 @@ type Context struct {
 type Secret struct {
 	Data map[string]string `json:"data"`
 }
+
+type PMM struct {
+	Enabled bool   `json:"pmm_enabled,omitempty"`
+	Image   string `json:"pmm_eimage,omitempty"`
```
Comment (idexter) [inline broker/service-instance.go]: Is this a typo `pmm_eimage`??

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:master-broker-PR-53-030d98f

**Review by cap1984 — APPROVED** [review]

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:master-broker-PR-53-f1f9dac

**Review by cap1984 — APPROVED** [review]

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:master-broker-PR-53-b108a99

**Review by dAdAbird — APPROVED** [review]

**Pull request merged**